test/framework/fixtures/libgoalFixture.go

```diff
@@ -311,6 +312,10 @@ func (f *LibGoalFixture) ShutdownImpl(preserveData bool) {
 	f.NC.StopKMD()
 	if preserveData {
 		f.network.Stop(f.binDir)
+		f.dumpLogs(f.PrimaryDataDir() + "/node.log")
```

Comment:
since we kind of unofficially support Windows, could you also use `filepath.Join(f.PrimaryDataDir(), "node.log")` instead here and below?